src/anemoi/training/utils/checkpoint.py

```diff
@@ -65,3 +98,37 @@ def save_inference_checkpoint(model: torch.nn.Module, metadata: dict, save_path:
     torch.save(model, inference_filepath)
     save_metadata(inference_filepath, metadata)
     return inference_filepath
+
+
+def transfer_learning_loading(model: torch.nn.Module, ckpt_path: Path | str) -> nn.Module:
+
+    # Load the checkpoint
+    try:
+        checkpoint = torch.load(ckpt_path, map_location=model.device)
+
+    # TODO @anaprietonem: this is a patch for issue #57
```

Comment:
I'd probably avoid putting TODOs in other people's names.

Reply:
This is just a note for Ana since she opened the issue. 
This way she will know to apply a change here when the issue is finally solved.
@anaprietonem Is this fine for you? :)